iohub/reader.py

```diff
@@ -80,7 +81,7 @@ def _check_multipage_tiff(src: str):
     return False
 
 
-def _check_pycromanager(src: str):
+def _check_ndtiff(src: str):
```

Comment:
I'm in favor of this renaming. Maybe consider changing `pycromanager` -> `ndtiff` everywhere else for consistency?

Reply:
Will do!